addOns/openapi/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Changed
 - Maintenance changes.
+- Promoted to Release status, with the migration of the core Spider (3113).
```

Comment:
Was added to other add-on.

Reply:
Oh crud, duh!

I must have had it open while commenting on that other PR :cry: